### ISSUE TITLE
Invalid #r syntax in Readme.me and test scripts

### ISSUE THREAD

**Issue by Sharpiro** [seesharper/csx#1]
Hi, I didn't realize you also had a script runner/debugger.  How does it differ from dotnet-script?

I wrote up a long post about an issue I was having before I realized I just needed to change a ```/``` to a ```,``` in the ```#r``` syntax.

This:
```
#r "nuget:AutoMapper/6.0.0"
```
Should be:
```
#r "nuget:AutoMapper,6.0.2"
```

Your main readme.md and a couple of your samples in build/temp seem to still be using the ```/``` instead.

**Comment by seesharper** [comment]
It is different in the sense that it does need a project file. The scripts are self contained meaning that references to NuGet packages are specified in the script itself. 
The philosophy here is that scripts are just scripts and should not require anything else than the script(s) itself


The latest version (2.0.0) also should also fix your issue with EF.

**Comment by Sharpiro** [comment]
Wow this is really great work!  I don't know how you figured this stuff out.

I tested with CSX 2.0 and it worked, but it is showing a few warnings.  Is there a way to suppress these, or do you think they are worth logging?  It just makes the output look a little messy.

Warnings:
```
warn: csx.ScriptExecutor[0]
      warning CS1701: Assuming assembly reference 'System.Linq.Expressions, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
' used by 'Microsoft.EntityFrameworkCore' matches identity 'System.Linq.Expressions, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f
11d50a3a' of 'System.Linq.Expressions', you may need to supply runtime policy
warn: csx.ScriptExecutor[0]
      C:\Users\me\Desktop\temp\csxgen\main.csx(7,13): warning CS1701: Assuming assembly reference 'System.Linq.Expressions, Version=4.0.1
0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'System.Linq.Queryable' matches identity 'System.Linq.Expressions, Version=4.1.
1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Linq.Expressions', you may need to supply runtime policy
warn: csx.ScriptExecutor[0]
      warning CS1701: Assuming assembly reference 'System.Linq.Expressions, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
' used by 'Microsoft.EntityFrameworkCore' matches identity 'System.Linq.Expressions, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f
11d50a3a' of 'System.Linq.Expressions', you may need to supply runtime policy
warn: csx.ScriptExecutor[0]
      C:\Users\me\Desktop\temp\csxgen\main.csx(7,13): warning CS1701: Assuming assembly reference 'System.Linq.Expressions, Version=4.0.1
0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'System.Linq.Queryable' matches identity 'System.Linq.Expressions, Version=4.1.
1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Linq.Expressions', you may need to supply runtime policy
```

**Comment by seesharper** [comment]
Thanks :) The main reason that it is even possible to figure out things like this, is that it is all open source.
I have just published CSX 2.0.2 which should take care of the warnings you see
